### PR TITLE
Extension now supports all VS19 versions, updated dependencies

### PR DESCRIPTION
**Pull request by HLCaptain**
Broadened VS19 support, now containing Pro and Enterprise versions beside Community. Updated dependencies. Builds only with VS19 and installable only for VS19. Visit `master` branch for 2.x version, which is compatible for VS22.

### CODE REVIEW

File: GodotAddinVS/GodotAddinVS.csproj [hunk 1]
```diff
@@ -78,10 +78,10 @@
     <PackageReference Include="Clide" Version="4.1.1" ExcludeAssets="runtime" />
     <PackageReference Include="Clide.Windows" Version="4.1.1" ExcludeAssets="runtime" />
     <PackageReference Include="GodotTools.IdeMessaging" Version="1.1.1" />
-    <PackageReference Include="Microsoft.VisualStudio.SDK" Version="16.0.205" ExcludeAssets="runtime">
+    <PackageReference Include="Microsoft.VisualStudio.SDK" Version="16.10.31321.278" ExcludeAssets="runtime">
       <IncludeAssets>compile; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.VSSDK.BuildTools" Version="16.5.2044">
+    <PackageReference Include="Microsoft.VSSDK.BuildTools" Version="16.11.35">
```
Comment (neikeq) [inline GodotAddinVS/GodotAddinVS.csproj:84]: Are these two upgrades necessary? Won't this prevent its use on VS19 installs that are still on previous minor versions? Specially `16.11.35` is only 3 months old.

File: GodotAddinVS/source.extension.vsixmanifest [hunk 2]
```diff
@@ -1,15 +1,32 @@
...
+        </InstallationTarget>
+        <InstallationTarget Version="[16.0,)" Id="Microsoft.VisualStudio.Community">
+
+        </InstallationTarget>
+        <InstallationTarget Version="[16.0,)" Id="Microsoft.VisualStudio.Pro">
+
+        </InstallationTarget>
+        <InstallationTarget Version="[16.0,)" Id="Microsoft.VisualStudio.Enterprise">
+
+        </InstallationTarget>
```
Comment (neikeq) [inline GodotAddinVS/source.extension.vsixmanifest:29]: The entries are duplicated.

Comment (HLCaptain) [reply GodotAddinVS/source.extension.vsixmanifest:29]: I will refactor it.

File: GodotAddinVS/GodotAddinVS.csproj (hunk 1 above)
Comment (HLCaptain) [reply GodotAddinVS/GodotAddinVS.csproj:84]: I can reverse this. Although I thought up to date dependencies, which are building with VS19 are cool. Having a robust system is always better if it has little to none negative effect on the end user.

Comment (neikeq) [reply GodotAddinVS/GodotAddinVS.csproj:84]: Yes, but we should give some room for upgrading. I think any version beyond 6 months old is fine.

**Comment by neikeq** [comment]
Thanks!

**Pull request merged**